### DISCUSSION TITLE
Restore Not Working

### DISCUSSION THREAD

**Discussion by tdashmike** [koush/scrypted#1281] [Q&A]
I tried updating with my own watchtower container and it failed after deleting the previous Scrypted v0.88.0 container. It had worked previously so I'm not sure what is going on, it may be because I updated docker yesterday. I recreated the contained with v0.89.0 but the backup file doesn't load and the website stops working until I restart the container. The container console doesn't appear to show any errors regarding the restore.

```
time="2024-01-23T01:54:46Z" level=info msg="Stopping /scrypted (7f7ad45bb55e) with SIGTERM"

time="2024-01-23T01:54:51Z" level=info msg="Creating /scrypted"

time="2024-01-23T01:54:51Z" level=error msg="Error response from daemon: invalid config for network host: invalid endpoint settings:\nnetwork-scoped alias is supported only for containers in user defined networks"
```

**Comment by koush** [comment]
>  it may be because I updated docker yesterday

Probably that.

**Comment by tdashmike** [comment]
> > it may be because I updated docker yesterday
> 
> Probably that.

Sorry, maybe I wasn't being clear. The docker update probably caused watchtower to fail but the restore doesn't work for me on a new installation on docker. It seems to load the file and pops up a window asking to reconnect but the container log doesn't show anything.

**Comment by koush** [comment]
Did you rerun the docker compose install script

**Reply by tdashmike** [reply]
Yes, I had to rebuild the container. It is the standard .yml that was made with the install script but I uncommented the part I needed for hardware acceleration.

**Comment by koush** [comment]
I'm unsure why the restore would fail. You can always unzip the restore file yourself and place it into the volume. See https://docs.scrypted.app/migration.html#manual-backup-and-restore

**Reply by tdashmike** [reply]
OK, I wasn't sure if the files can be placed directly in the volume folder. I'll try it when I get back home. Is there a way I can create a log for you for the restore function?

**Reply by tdashmike** [reply]
Does this error have anything to do with the backup not loading? The newly created container will generate a backup and if I restore the backup that was just created nothing happens. Below I am showing that I'm copying the backup zip contents into the volume folder, same result. When I start the container it will ask me to create a login.

```
root@t-server:~/.scrypted/volume# cp -r /home/m/scrypted_backup/. ~/.scrypted/volume/
root@t-server:~/.scrypted/volume# ls
000003.log  000004.log  000005.ldb  CURRENT  LOCK  LOG  MANIFEST-000002
```

```
i WebRTC Plugin loaded @scrypted/webrtc
(node:115) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
no device was returned by the plugin 20
node modules /server/volume/plugins/@scrypted/webrtc/node18-linux-x64-20241303
(node:115) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
```
